gulpfile.js

```diff
@@ -73,11 +73,20 @@ var paths = {
 			'./browser/vendor/three/MorphAnimMesh.js',
 			'./browser/vendor/three/Font.js',
 			'./browser/vendor/three/FontLoader.js',
-		
+			'./browser/vendor/three/loaders/FBXLoader.js',
+			'./browser/vendor/three/loaders/ColladaLoader2.js',
+			'./browser/vendor/three/loaders/gltf/glTF-Parser.js',
+			'./browser/vendor/three/loaders/gltf/glTFAnimation.js',
```

Comment:
these are going to make the player *a lot* bigger, aren't they? what if we used the Core.load_aux_script() method for these, and only loaded them when the corresponding loader/asset is used?